pages/index.js

```diff
@@ -26,16 +27,19 @@ export const getStaticProps = async () => {
 export default function Home({ image }) {
   return (
     <PrimaryLayout image={image} inNeed main>
-      <HotMealDaySection />
+      {/* <HotMealDaySection /> */}
```

Comment:
Why did you remove this section?

Reply:
They were separate components and have similar styles so, I have created a `cardsData.json` and map over it.

Reply:
Let's put this section back as it's part of the design.

Reply:
```suggestion
      <HotMealDaySection />
```